main.go

```diff
@@ -564,15 +564,20 @@ This command is meant to be used from the boot GRUB menu, but can likely be used
 				Name:  "unattended",
 				Usage: "Do not wait for user input and provide ttys after reset. Also sets the fast mode (do not wait 60 seconds before reset)",
 			},
+			&cli.BoolFlag{
```

Comment:
this is a new flag to expose the underlying config. We allowed this via config or event but not via manual flags..